examples/CFD/airfoil3d.py

```diff
@@ -34,6 +64,8 @@ def nacaAirfoil(x, thickness, chordLength):
     return domain
 
 class Airfoil(KBCSim):
+    def __init__(self, **kwargs):
```

Comment:
These two lines for __init__ and super should not appear in any of the examples. All parent classes should do this initialization. Please remove these every where

Reply:
I disagree with this. The inclusion of an __init__ method in the subclass and invoking the superclass's __init__ method ensures proper handling of arguments before the superclass is invoked and is generally a good/common practice. Additionally, we might need to add state variables to our class. If we intend to use these state variables in the boundary_condition constructor, we must set these states before calling super.

Currently, in some of our examples, we are improperly defining variables in the main function (such as u_wall) and using them within the class. This is a non-standard and incorrect practice that we need to address in the future.

Please refer to this StackOverflow post for more detailed information: https://stackoverflow.com/a/52285154/8466369"

Reply:
I understand that these basics. I think it creates unnecessary complications for the novice user. So I suggested removing it for simplicity. For u_wall and other example specific variables they can be defined in the same example. Either in main or inside each function.  u_wall for example should not be part of self as it is not used in any of the core computations and is just for problem setup and definition.

Reply:
Also: explicit is better than implicit.

Reply:
Copying the class to a different file would never work simply because it needs some other modules and classes to be imported. So there is no logic behind a class being self contained. You can also declare a variable inside the class and not assign it to self.

Reply:
Exactly!!!! And for the same exact reason I mentioned you don't import the other modules in main, although if you do (except for the KBCSim), everything works!!